src/py/migraphx_py.cpp

```diff
@@ -272,6 +272,14 @@ MIGRAPHX_PYBIND11_MODULE(migraphx, m)
             py::arg("op"),
             py::arg("args"),
             py::arg("mod_args") = std::vector<migraphx::module*>{})
+        .def(
+            "add_literal",
+            [](migraphx::module& mm, const migraphx::shape& s, py::buffer data) {
```

Comment:
There is no need to pass the shape as we can get it from the python buffer. You can use the `to_shape` function to convert the buffer to a shape.

Reply:
Made that change. but only concern is that API signature for `add_literal` for both python and c++ would differ.